docs/source/llms/llm-evaluate/index.rst

```diff
@@ -0,0 +1,521 @@
+    * `perplexity <https://huggingface.co/spaces/evaluate-metric/perplexity>`_ :sup:`1`
+    * `toxicity <https://huggingface.co/spaces/evaluate-measurement/toxicity>`_ :sup:`1`
+    * `ari_grade_level <https://en.wikipedia.org/wiki/Automated_readability_index>`_ :sup:`2`
+    * `flesch_kincaid_grade_level <https://en.wikipedia.org/wiki/Flesch%E2%80%93Kincaid_readability_tests#Flesch%E2%80%93Kincaid_grade_level>`_ :sup:`2`
+
+* **text models**:
+
+    * `perplexity <https://huggingface.co/spaces/evaluate-metric/perplexity>`_ :sup:`1`
+    * `toxicity <https://huggingface.co/spaces/evaluate-measurement/toxicity>`_ :sup:`1`
+    * `ari_grade_level <https://en.wikipedia.org/wiki/Automated_readability_index>`_ :sup:`2`
+    * `flesch_kincaid_grade_level <https://en.wikipedia.org/wiki/Flesch%E2%80%93Kincaid_readability_tests#Flesch%E2%80%93Kincaid_grade_level>`_ :sup:`2`
+
+
+:sup:`1` Requires `evaluate <https://pypi.org/project/evaluate>`_, `pytorch <https://pytorch.org/get-started/locally/>`_, and 
+`transformers <https://huggingface.co/docs/transformers/installation>`_
+
+:sup:`2` Requires `textstat <https://pypi.org/project/textstat>`_
+
+:sup:`3` Requires `evaluate <https://pypi.org/project/evaluate>`_, `nltk <https://pypi.org/project/nltk>`_, and 
+`rouge-score <https://pypi.org/project/rouge-score>`_
+
+However, using the pre-defined metrics associated with a given model type is not the only way to generate scoring metrics 
+for LLM evaluation in MLFlow. MLflow provides two ways for selecting metrics to evluate your LLM:
+
+1. Specify the ``model_type`` argument in :py:func:`mlflow.evaluate` 
+
+    * Each predefined model type comes with a standard set of metrics that are available for relevant evaluation of a model type. 
+    * The defaults are suitable if your model falls in one of the predefined categories (e.g., ``question-answering``).   
+
+An example of using the predefined metrics for a given ``model_type`` is shown below:
+
+.. code-block:: python
+
+    results = mlflow.evaluate(
+        model,
+        eval_data,
+        targets="ground_truth",
+        model_type="question-answering",
+    )
+
+1. Specify a custom list of metrics by explicitly referencing a metric calculation function.
+
+    * To add additional metrics to the default collection from part 1 above, add the function names to the ``extra_metrics`` argument.
+    * To diable default metric calculation and only calculate explicit metrics, remove the ``model_type`` argument and define the desired metrics. 
+
+An example of disabling the default metrics and explicitly declaring a subset of metrics to calculate is shown below:
+
+.. code-block:: python
+
+    results = mlflow.evaluate(
+        model,
+        eval_data,
+        targets="ground_truth",
+        extra_metrics=[mlflow.metrics.toxicity(), mlflow.metrics.latency()],
+    )
+
+
+The full reference for supported evaluation metrics can be found `here <../python_api/mlflow.html#mlflow.evaluate>`_. 
+
+
```

Comment:
@chenmoneygithub we need to add a section here where we discuss the pre-canned LLM judged metrics such as faithfulness and answer_similarity. can we add that here?

Reply:
sure!